.clang-format

```diff
@@ -11,3 +11,4 @@ BreakInheritanceList: BeforeComma
 KeepEmptyLinesAtTheStartOfBlocks: false
 PointerAlignment: Left
 SpaceAfterTemplateKeyword: false
+IndentRequires: false
```

Comment:
I'm a bit on the fence with this one, not sure if I like it indented or not.

Reply:
All existing `requires` in the code are not indented.